src/api.ts

```diff
@@ -0,0 +1,28 @@
+const API_TODOS = 'https://jsonplaceholder.typicode.com/todos';
+const API_USERS = 'https://jsonplaceholder.typicode.com/users';
+
```

Comment:
you can use one constant and add where necessary

export const API_BASE = 'https://jsonplaceholder.typicode.com';&nbsp;
export const API_USERS = '/users';&nbsp;
export const API_TOODS = '/todos';&nbsp;